src/python/pants/pantsd/pants_daemon.py

```diff
@@ -195,4 +195,7 @@ def pre_fork(self):
 
   def post_fork_child(self):
     """Post-fork() child callback for ProcessManager.daemonize()."""
+    for service in self._services:
+      service.post_fork()
```

Comment:
This sounds like a good idea. And by doing this, set panic handler once will cover both pantsd and daemon worker process, as they share 1 Native object (or forked from the shared object).